frontend/amundsen_application/static/js/pages/TableDetailPage/WatermarkLabel/constants.ts

```diff
@@ -1,5 +1,5 @@
-export const NO_WATERMARK_LINE_1 = 'No valid partitions found';
-export const NO_WATERMARK_LINE_2 = 'Unknown date range';
+export const NO_WATERMARK_LINE_1 = 'Not Available';
+export const NO_WATERMARK_LINE_2 = '';
```

Comment:
this one by default shows:
```
Date Range
  No valid partitions found
  Unknown date range
```
which is kind confusing.

let's just update the text to "Not Available"